themes/janeswalk/js/components/pages/WalkPage/WalkAccessibility.jsx

```diff
@@ -1,21 +1,13 @@
-import React from 'react';
-
-const WalkAccessibility = ({checkboxes}) => {
+const WalkAccessibility = ({checkboxes, accessibility, style}) => {
 
   const accessibilityKeys = Object.keys(checkboxes).filter(item => item.includes("accessible"));
 
-  const accessibility = {
-    "accessible-familyfriendly":"Family friendly",
-    "accessible-seniors":"Senior Friendly",
-    "accessible-busy":"Busy sidewalks",
-  };
-
   return (
-    <section className="walkAccessibility">
-      <a name="Accessibility"></a>
+    <section className={`walkAccessibility ${style}`}>
+      {style === 'walk-page' ? <a name="Accessibility"></a> : ''}
```

Comment:
I think (but am not sure that) `null` is preferable to `''`, as `''` is still typeof string. React sees strings and wraps `<span>`s around them according to some hard to anticipate rules, while `null` just inserts nothing.